exercises/practice/leap/.approaches/boolean-chain/content.md

```diff
@@ -0,0 +1,94 @@
+# Chain of boolean expressions
+
+```cpp
+bool is_leap_year(int year) {
+    return (year % 4 == 0) && (year % 100 != 0 || year % 400 == 0);
```

Comment:
very tiny nitpick: You could either omit the first pair of parentheses:
`return year % 4 == 0 && (year % 100 != 0 || year % 400 == 0);`
or add parentheses to the second and third sub-expression, like in the "shortened" example below:
`(year % 4 == 0) && ((year % 100 != 0) || (year % 400 == 0));`